src/utils/LaunchUtils.ts

```diff
@@ -0,0 +1,70 @@
+import Profile from "../model/Profile";
+import R2Error from "../model/errors/R2Error";
+import Game from "../model/game/Game";
+import FsProvider from "../providers/generic/file/FsProvider";
+import GameRunnerProvider from "../providers/generic/game/GameRunnerProvider";
+import GameDirectoryResolverProvider from "../providers/ror2/game/GameDirectoryResolverProvider";
+import ManagerSettings from "../r2mm/manager/ManagerSettings";
+import ModLinker from "../r2mm/manager/ModLinker";
+
+export const launch = async (game: Game, profile: Profile, modded: boolean): Promise<void> => {
```

Comment:
Bit nitpicky and probably up to @MythicManiac but a class approach with static methods is probably nicer to easily see where code is scoped. Currently references look like they’d be defined internally inside files that use this.

Reply:
I'm a bit split on this. I like classes for clarity, but I don't like classes because they cause more indents and tend to lead to less pure functions (as you easily end up with some reliance on class state).

A single file is technically a module already, but you could always export an object even if the definitions aren't in a class, giving you best of both worlds.